public/styles.css

```diff
@@ -1,3 +1,61 @@
-body {
-  margin: 0;
+#main {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+.mainBox {
+  flex-direction: column;
+  /*align-items: center;*/
+  justify-content: space-around;
+  display: flex;
+  flex-wrap: wrap;
+  align-self: flex-start;
+}
+.dtborder {
+  position: relative;
+  border: 20px solid #ffecb7;
+  background: #ffffff;
+  padding: 60px;
+  text-align: center;
+  height: 100px;
+  line-height: 100px;
```

Comment:
czemu line-height? Line height to właściwość określająca interlinie. Tutaj nam to nie potrzebne chyba?